backend/dataall/modules/s3_datasets/cdk/dataset_stack.py

```diff
@@ -437,20 +449,68 @@ def __init__(self, scope, id, target_uri: str = None, **kwargs):
             },
         )
 
+        glue_sec_conf_enc_key = kms.Key(
+            self,
+            f'crwlr_log_enc_key_{dataset.GlueCrawlerName}',
+            removal_policy=RemovalPolicy.DESTROY,
+            alias=f'crwlr_log_enc_key_{dataset.GlueCrawlerName}',
```

Comment:
can we have both `alias` and `kms.Key` construct id be  `{dataset.GlueCrawlerName}_log_enc_key`

that way the naming convention is prefixed with `resource_prefix`

nit: we can also set it once as variable and pass that as both construct id and alias name